client/app/components/onboardingGuide/GuideCoverPhotoPage.js

```diff
@@ -0,0 +1,51 @@
+import { PropTypes } from 'react';
+import r, { div, h2, p, img, a } from 'r-dom';
+import css from './styles.scss';
+
+import GuideBackToTodoLink from './GuideBackToTodoLink';
+
+const GuideCoverPhotoPage = (props) => {
+  const { changePage, initialPath, t, pageData, infoIcon } = props;
+
+  return div({ className: 'container' }, [
+    r(GuideBackToTodoLink, { changePage, initialPath, t }),
+    h2({ className: css.title }, t('title')),
+    p({ className: css.description }, t('description')),
+
+    pageData.info_image ?
+      div({ className: css.sloganImageContainer }, [
+        img({
+          className: css.sloganImage,
+          src: pageData.info_image,
+          alt: t('info_image_alt'),
+        }),
+      ]) :
+    null,
```

Comment:
I guess, this should be indented more.
('if's become quite hard to follow with this style)